live-ingester/pom.xml

```diff
@@ -32,16 +36,18 @@
 			<artifactId>camel-activemq-starter</artifactId>
 		</dependency>
 		<dependency>
-			<groupId>org.apache.camel.springboot</groupId>
-			<artifactId>camel-sjms-starter</artifactId>
+			<groupId>org.apache.camel</groupId>
+			<artifactId>camel-http</artifactId>
```

Comment:
You can use here:
`<dependency>
	<groupId>org.apache.camel.springboot</groupId>
	<artifactId>camel-http-starter</artifactId>
</dependency>`
which already comes with `httpclient` inside

Reply:
I think it is not camel that needs it, but `SignedStorageClient` and it previously was supplied as some transitive dependency - so it is better to make direct dependency rather that rely on another transitive one

Reply:
Ok, `SignedStorageClient` won't need it any more, but I can remove it in my PR. LGTM